src/app/components/authorized/welcome-page/welcome-page.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../../colors.scss';
+@import '../../../../mixins.scss';
+
+.gridWrapper {
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
+  grid-template-rows: 1fr auto;
+  row-gap: 2em;
+  grid-template-areas: 
+    "navigationBar navigationBar navigationBar"
+    ". mainContent .";
+}
+
+.navigationBar {
+  grid-area: navigationBar;
+}
+
+.mainContent {
+  grid-area: mainContent;
+  display: grid;
+  grid-template-rows: 1fr 1fr;
+  grid-template-areas: 
+    "headerPanel"
+    "sidePanel";
+  @include boxshadow;
+}
+
+.headerPanel {
+  grid-area: headerPanel;
+  display: grid;
+  grid-template-rows: 1fr;
+  grid-template-areas: 
+    "welcomeSign";
+  background-color: $darkBlue;
+  color: $lightTextColor;
+  text-transform: uppercase;
+  text-align: center;
+  align-self: stretch;
+  padding: 6rem;
+  border-left: 1rem solid $orange;
+}
+
+.welcomeSign {
+  grid-area: welcomeSign;
+  font-size: 2,5em;
+  letter-spacing: 0.5rem;
+  strong {
+    color: $orange;
+  };
+  text-shadow: 1px 1px 4px black;
+}
+
+.sidePanel{
+  grid-area: sidePanel;
+  background-color: $lightTextColor;
+  color: $darkBlue;
+  padding: 2.5rem;
+  display: grid;
+  row-gap: 1rem;
+  > div {
+    align-items: center;
+    > h2 {
+        grid-area: sidePanelMessage;
+    };
+    > button {
+      grid-area: subBtn;
+      @include button;
+      justify-self: center;
+    };
+    display: grid;
+    grid-template-columns: 4fr 1fr;
+    column-gap: 1rem;
+    grid-template-areas: 
+      "sidePanelMessage subBtn";
+  };
+}
```

Comment:
I think some of that should be at the app-root component. Since landing page is really just for new useres. (navigationBar for exapmle)

Reply:
No, the same problem as before, forgot to solve this problem after creating authorizedComponent.